package.json

```diff
@@ -106,6 +106,7 @@
     "city-timezones": "^1.2.1",
     "eslint": "^8.34.0",
     "lucide-react": "^0.171.0",
+    "moment-timezone": "^0.5.43",
```

Comment:
lets not add another timezone package. i think dayjs does this already

Reply:
Sure! We'll modify this; we will take a look at the hint Alishaz provided.

Reply:
> lets not add another timezone package. i think dayjs does this already

We already have a helper utility function that handles this quite well 🙏 I've provided the reference and @DrDrewCain should be able to use that 🎉